Makefile

```diff
@@ -165,8 +158,8 @@ cmd/logcli/logcli: $(APP_GO_FILES) cmd/logcli/main.go
 # Loki #
 ########
 
-loki: protos yacc ragel cmd/loki/loki
-loki-debug: protos yacc ragel cmd/loki/loki-debug
+loki: cmd/loki/loki
+loki-debug: cmd/loki/loki-debug
```

Comment:
we don't need to build protos, yacc, ragel to build loki, if someone is changing those things they will need to know how to run make protos, make yacc etc